CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 5.3.0 (2016-2-19)
```

Comment:
Sorry, I somehow missed your last comment, that was my bad.

OK, I think I worked out a solution for you:

```
# Save your current work on pingread
git checkout -b temp

# Go back to master
git checkout master

# DANGER: this will delete all of your work off of master. If anything at all goes wrong, stop here and report back. Your work will be saved on branch temp.
git reset --hard f750377e

# If you've already added my repo as a remote, feel free to skip
git remote add nebrius git@github.com:nebrius/raspi-io.git

# If you already added my repo as a remote, use the name you gave it here instead of nebrius
git pull nebrius master

# This will remove your work from your master branch on GitHub and add in my latest changes
git push --force origin master

# This will be the branch that is used for the PR
git checkout -b pingread

# This will pull your branches from temp, but without my commits in the middle giving a clean history
git cherry-pick --strategy=recursive -X theirs f713bb0e 903286ed 5a273eb7 c01f2685

# Push the branch
git push origin pingread
```

After you've done those steps, open a new PR from that branch and close this one, and everything should be good!